examples/example1/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
I don't think you really need any of this.

Reply:
Removed.